src/Bedrock.Framework.Experimental/Protocols/Tls/ApplicationLayerProtocolType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls
+{
+    internal enum ApplicationLayerProtocolType
+    {
+        None = -1,
+        Http1_1 = 0,
+        Spdy1 = 1,
+        Spdy2 = 2,
+        Spdy3 = 3,
+        Turn = 4,
+        Stun = 5,
+        Http2_Tls = 6,
+        Http2_Tcp = 7,
```

Comment:
What is this? Where do these values come from? Any urls?

Reply:
Alpn so you can tell if it's http1.1 http/2 etc during negotiation there is an Iana registry page for it that I will update it with